modules/porous_flow/doc/tests/radialinjection.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import matplotlib.pyplot as plt
+import pandas as pd
```

Comment:
I do not think we distribute `pandas` in our MOOSE package. However, I understand why you used it. It is way more elegant than writing your own function for it... I did that in RELAP and I wished I had something better available. So, maybe we should add `pandas` to our package. Tagging @permcody, @milljm